docs/logical-data-model.md

```diff
@@ -0,0 +1,80 @@
+# Logical data model
+
+![Resource](./assets/logical-data-model-1.png)
+
+## Resource
+
+| Key | Value description |
+| --- | ----------------- |
+| `id` | A unique resource id. |
+| `parentId` | The parent resource id. |
+| `version` | Resource version. |
```

Comment:
Is this the version of the resource or the resource specification? I think it is the resource, i.e. that we are versioning the resource request, but just wanted to understand.

Reply:
Now separate fields;
- specificationVersion
- resourceVersion